tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -1418,16 +1418,27 @@ <h2>Show similarity to selected datapoint</h2>
                               Show nearest counterfactual datapoint
                             </paper-toggle-button>
                           </div>
-                          <paper-radio-group
-                            selected="{{nearestCounterfactualDist}}"
+                          <template
+                            is="dom-if"
+                            if="[[customDistanceFunctionSet]]"
```

Comment:
is this if statement backwards?

Reply:
the commits are not complete yet. I will let you know when it is done.